sys/parsoid.js

```diff
@@ -159,11 +159,27 @@ function _dependenciesUpdate(hyper, req) {
     });
 }
 
+function compileReRenderBlacklist(blacklist) {
+    const result = {};
+    blacklist = blacklist || {};
+    Object.keys(blacklist).forEach((domain) => {
+        result[domain] = blacklist[domain].map((entry) => {
```

Comment:
Minor optimization opportunity: You could build up a single regexp per domain with `|`.